tiledb/sm/filesystem/s3.cc

```diff
@@ -49,6 +53,26 @@
 
 #include "tiledb/sm/filesystem/s3.h"
 
+namespace {
+
+Aws::Utils::Logging::LogLevel aws_log_name_to_level(std::string loglevel) {
+  if (loglevel == "Fatal")
```

Comment:
Can you add a case normalization step here? Pretty sure we'd run into that at some point. e.g.

```
std::transform(loglevel.begin(), loglevel.end(), loglevel.begin(), ::tolower);
```